ibis/tests/expr/test_window_functions.py

```diff
@@ -73,3 +73,19 @@ def test_rank_followed_by_over_call_merge_frames(alltypes):
     expr1 = t.f.percent_rank().over(ibis.window(group_by=t.f.notnull()))
     expr2 = ibis.percent_rank().over(group_by=t.f.notnull(), order_by=t.f)
     assert expr1.equals(expr2)
+
+
+def test_duplicate_order_by_errors():
+    """If you specify the order_by argument multiple times, it should raise an error."""
```

Comment:
It's not clear to me this should be the behavior. It seems like it would be equally valid to combine the order bys.

This change here seems unrelated to the goal of the PR, unless I'm misunderstanding something. I'll remove this in a separate commit, and we can discuss these changes in a separate PR.

Reply:
This was me being conservative: I didn't know what we wanted the behavior to be, so I didn't want people to rely upon anything, and this leaves the door open for us to choose whatever we want later.

There is some other issue in our bug tracker if you did two consecutive .order_by()s on a table before Ibis would combine them, but then we decided that this had bad semantics and so we made it so the final call overrode all previous calls. I didn't want to repeat that breaking change.

Reply:
But your plan sounds fine to me, this doesn't seem like a huge deal, I'd love to get this fix in and then sort out this detail later

Reply:
Oh, [here is that PR](https://github.com/ibis-project/ibis/pull/8618/files#diff-ab670a40760203d7666d8e07cc34a6a9cc06cf887706c41defaff7232f2452b5), and it looks like we actually DO merge order bys. But I still think that is not the best behavior.